lib/Parse/ParsePattern.cpp

```diff
@@ -431,6 +431,13 @@ mapParsedParameters(Parser &parser,
                            param.FirstName, param.FirstNameLoc);
     }
 
+    // Warn when an unlabeled parameter follows a variadic parameter
+    if (ellipsisLoc.isValid() && elements.back()->isVariadic() &&
+        param.FirstName.empty()) {
```

Comment:
This example looks well-formed. So long as the parameter following the variadic is labeled, subsequent parameters can also be unlabeled because their arguments must follow the labeled parameter. (This is one of the nicer consequences of [SE-0060](https://github.com/apple/swift-evolution/blob/master/proposals/0060-defaulted-parameter-order.md))